src/main/java/com/stripe/model/Subscription.java

```diff
@@ -856,6 +856,13 @@ public Discount deleteDiscount(Map<String, Object> params, RequestOptions option
     @SerializedName("konbini")
     Konbini konbini;
 
+    /**
+     * This sub-hash contains details about the ACH direct debit payment method options to pass to
+     * invoices created by the subscription.
+     */
+    @SerializedName("us_bank_account")
+    Invoice.PaymentMethodOptions.UsBankAccount usBankAccount;
```

Comment:
Is it expected we re-use the type here? I thought we stopped "sharing" types in java a while ago